pyproject.toml

```diff
@@ -39,7 +39,7 @@ mypy = "1.15.0"
 pandas = "2.2.3"
 pyarrow = ">=10.0.1"
 pytest = ">=7.1.2"
-pyright = ">=1.1.396,!=1.1.398"
+pyright = "1.1.396"
```

Comment:
Locking in the version here just to check that CI is passing, will be reverted once #1185 is merged.

Reply:
OK - this is a good idea.